### ISSUE TITLE
Feature request: pd-mark-as-unplayed

### ISSUE THREAD

**Issue by rnkn** [akrennmair/newsbeuter#588]
**Newsbeuter version (copy from `newsbeuter -v`)**:
newsbeuter 2.10-69a5
System: Darwin 16.7.0 (x86_64)
Compiler: g++ 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.42)
ncurses: ncurses 5.7.20081102 (compiled with 5.7)
libcurl: libcurl/7.54.0 SecureTransport zlib/1.2.8 (compiled with 7.51.0)
SQLite: 3.19.3 (compiled with 3.19.3)
libxml2: compiled with 2.9.4

Steps to reproduce the issue:

1. open podbeuter
2. press p on selected podcast
3. podcast will be marked as played with no obvious way to mark as unplayed ("ready")

**Other info you think is relevant**:

I'd like a command (`pb-mark-as-unplayed`?) to return the selected podcast to unplayed status 😃

**Comment by Minoru** [comment]
Thanks for the suggestion; podbeuter is kinda abandoned at the moment, though, so don't get your hopes of seeing that implemented soon too high; sorry about that.

**Comment by rnkn** [comment]
Hello, I no longer use newsboat/newsbeuter for podcasts so if I was the only person requesting this feature, do you want to close this?

**Comment by Minoru** [comment]
No, I still think it makes sense to have this function, since it parallels `toggle-read` in Newsboat/Newsbeuter.

Sorry this wasn't implemented in time to be useful to you. What program did you switch to?

**Comment by rnkn** [comment]
I just don't really listen to podcasts. Still use newsboat for RSS.
